cordformation/src/test/resources/net/corda/plugins/DeployDockerImage.gradle

```diff
@@ -0,0 +1,31 @@
+buildscript {
+    ext {
+        corda_group = 'net.corda'
+        corda_release_version = '4.0'
+    }
+}
+
+plugins {
+    id 'net.corda.plugins.cordformation'
+}
+
+
+
+repositories {
+    mavenCentral()
+    maven { url 'https://software.r3.com/artifactory/corda-releases' }
+
+}
+
+dependencies {
+    runtime "$corda_group:corda:$corda_release_version"
+    runtime "$corda_group:corda-node-api:$corda_release_version"
```

Comment:
Can these be `runtimeOnly` instead of `runtime` please? The `runtime` configuration is deprecated.